sdk/core/azure-core-amqp/test/ut/claim_based_security_tests.cpp

```diff
@@ -151,9 +165,8 @@ namespace Azure { namespace Core { namespace Amqp { namespace Tests {
   TEST_F(TestCbs, CbsOpenNoListener)
   {
     ConnectionOptions options;
-    MessageTests::AmqpServerMock mockServer;
     options.EnableTrace = true;
-    options.Port = mockServer.GetPort();
+    options.Port = GetPort() + 10;
```

Comment:
why "+ 10"?

Reply:
It's a number that isn't the port which is configured. The point of this test is to try to open a CBS operation where there's nobody listening on the port.

Reply:
Got it, that's what I assumed, it's just that I saw that the previous version of the test did not have a "+ 10" and was using the same port. Thanks for the comment in the code now!

Reply:
The previous version didn't actually have a listener present - but the new test is designed around either the Test AMQP broker or the mock server, and in the case of the Test AMQP broker, the server *will* be listening. So I needed to play games with the port.